### DISCUSSION TITLE
Partial Dependence Ploting isn't ploting

### DISCUSSION THREAD

**Discussion by poletto-matheus** [google/yggdrasil-decision-forests#153] [Q&A]
![PDP](https://github.com/user-attachments/assets/16f11a91-24a9-464d-85a0-9c08ddcada33)

Could you help me? It seems the model analyze it but I can't see!

**Comment by rstz** [comment]
Could this be the same issue as https://github.com/google/yggdrasil-decision-forests/issues/121 ?
